code/__DEFINES/_helpers.dm

```diff
@@ -49,3 +49,6 @@
 /// A null statement to guard against EmptyBlock lint without necessitating the use of pass()
 /// Used to avoid proc-call overhead. But use sparingly. Probably pointless in most places.
 #define EMPTY_BLOCK_GUARD ;
+
+/// Abstraction over using mob.client to just check if there's a connected player.
+#define HAS_CONNECTED_PLAYER(mob) (mob.client)
```

Comment:
is this just to put a name to it? or do you have other intentions with the macro

Reply:
Put a name to it. Since
```
if(client)
    persistent_client.thing()
```
is a pattern that would look strange, so abstracting "Has a connected user" made sense to me